src/app/static/src/components/downloadIndicator/DownloadIndicator.vue

```diff
@@ -33,14 +34,14 @@
         downloadingIndicator: boolean
     }
 
-    export default defineComponent<unknown, Methods, Computed, Props>({
```

Comment:
why have these type qualifiers been removed, whereas in other components they remain?